src/pages/ResearcherProfile.js

```diff
@@ -231,7 +219,8 @@ export default function ResearcherProfile(props) {
   };
 
   const eraValidate = async () => {
-    // Temporary fix until eRACommons.js is updated to share re-render info.
+    // This function is currently unused but can check if a user's eRA Commons ID is in date.
+    // It will be used to update the researcherIncompleteFields component when eRACommons.js shares re-render info.
```

Comment:
If this is unused, it should be removed. Unused code triggers console errors.

Reply:
It looks like it will have to be re-implemented when fixing the researcher profile page.